magento/magento2
# GraphQL CORS Headers

### Description (*)
As a web developer, I want Magento to support the CORS standard so that I can develop my applications according to defined best practices.

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#28561

### Manual testing scenarios (*)
1. go to admin > stores > configuration > services > graphql
2. enable CORS headers
3. Fill fields with data
4. save and clean cache
5. run graphQL query and confirm headers are added

### Questions or comments


### Contribution checklist (*)
